localstack/services/iam/iam_listener.py

```diff
@@ -28,6 +31,10 @@ def return_response(self, method, path, data, headers, response):
             MessageConversion.fix_date_format(response)
             MessageConversion.fix_error_codes(method, data, response)
             MessageConversion.fix_xml_empty_boolean(response, BOOL_ATTRS)
+
+            if AWS_SDK_JS in headers.get("User-Agent", ""):
+                MessageConversion.booleans_to_lowercase(response, BOOL_ATTRS)
```

Comment:
Wondering if we should change this for any type of SDK - lowercase booleans is broadly the accepted spelling in most cases (and should work across different AWS SDKs). We can pick this up in a follow-up PR, though (no need to change the PR at this point).

Reply:
I just tested it and it seems to fine work for python and nodejs clients.